tests/console/download_asset.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Downloads ASSET_1 file to specified by test_data file location.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler "get_test_suite_data";
+use File::Basename;
+
+sub run {
+    select_console 'root-console';
+    my $file_location    = get_test_suite_data()->{file_location};
```

Comment:
we could use `logs_path` as well in test data, should be easier to read.wdyt?

Reply:
The module can be used for general purposes, not just logs.